docs/based-applications/developers/based-application-development.md

```diff
@@ -0,0 +1,186 @@
+      }
+    ],
+    "validatorBalanceWeight": 0.3
+  },
+  {
+    "id": "2",
+    "tokenWeights": [
+      {
+        "token": "0x68a8ddd7a59a900e0657e9f8bbe02b70c947f25f",
+        "weight": 0.07321594068582021
+      }
+    ],
+    "validatorBalanceWeight": 0.7
+  }
+]
+```
+
+In this example, strategy number `2` has a risk-adjusted weight of `~0.07` for the specified token, but it has a validator balance weight of `0.7`.
+
+Strategy number `10`, has a much bigger risk-adjusted weight of `~0.93` for specified token, but it has a validator balance weight of `0.3`.
+
+For more information on how these risk-adjusted strategy-token weights are calculated, [please visit this page](../learn/based-applications/strategy-weights.md).
+
+### Final weight
+
+To combine data from the previous step into the final Weight it is necessary to use a **combination function**. Such function has to be defined by the bApp itself and can be tailored to its specific needs. Traditionally, one good combination functions include the arithmetic mean, geometric mean, and harmonic mean.
+
+Let's consider the bApp from step 1, which uses tokens **`0x68a8ddd7a59a900e0657e9f8bbe02b70c947f25f`** and **validator balance**. For the purpose of our example, this bApp considers **`0x68a8ddd7a59a900e0657e9f8bbe02b70c947f25f`** to be twice times as important as **validator balance**. 
+
+:::info
+Note: this is a purely fictional scenario, to show the strong impact of the coefficients attributed to tokens and validator balance, with respect to the final combined weight of a strategy.
+:::
+
+Below it's reported a code snippet, showing how to combine weights from step 1 using a simple arithmetic weighted average in the described scenario:
+
+```ts
+const validatorImportance = 1;
+const ssvTokenImportance = 2;
+
+console.info(`Using arithmetic weighted average to calculate Strategy weights.
+Validator Balance is 2 times more important than 0x68a8ddd7a59a900e0657e9f8bbe02b70c947f25f`);
+
+let simpleAverageStrategyWeights = new Map();
+for (const strategy of strategyTokenWeights) {
+  // calculate the strategy weight, combining token weight and validator balance weight
+  let strategyWeight =
+    ((strategy.validatorBalanceWeight || 0) * validatorImportance +
+      strategy.tokenWeights[0].weight * ssvTokenImportance) /
+    (validatorImportance + ssvTokenImportance);
+  // set the value in the mapping
+  simpleAverageStrategyWeights.set(strategy.id, strategyWeight);
+}
+
+console.info(
+  `Final Strategy weights: ${JSON.stringify(
+    Object.fromEntries(simpleAverageStrategyWeights),
+    undefined,
+    2
+  )}`
+);
+```
+
+This returns a map with the combined risk-adjusted weight of each strategy.
+
+```json
+Using arithmetic weighted average to calculate Strategy weights.
+Validator Balance is 2 times more important than 0x68a8ddd7a59a900e0657e9f8bbe02b70c947f25f
+Final Strategy weights: {
+  "2": 0.28214396045721346,
+  "10": 0.7178560395427865
+}
+```
```

Comment:
I assume this looks ok because it's not valid json?

Reply:
Noticed the same thing I should have fixed it afterwards :O